README.md

```diff
@@ -1 +1,154 @@
-# traffic-state-estimation
\ No newline at end of file
+# Traffic State Estimation Applications for Eclipse MOSAIC
+
+This repository contains applications for Floating Car Data (FCD) based Traffic State Estimation (TSE) to be used with
+[Eclipse MOSAIC](https://eclipse.dev/mosaic).
+Included metrics are partly based on a paper by Yoon et al.[^1].
+Furthermore, we provide additional MOSAIC configuration files to be used with the 
+[BeST Scenario](https://github.com/mosaic-addons/best-scenario)[^2], to reproduce and validate results and develop your own TSE metrics.
+
+## Documentation
+<p align="center"><img width="50%" src=".github/fcd-system-overview.png"/></p>
+The picture above depicts the fundamental structure and functioning of this application suite.
+The functionality is split across two applications `TseServerApp` and the `FcdTransmitterApp`.
+The `FcdTransmitterApp` is a vehicle application that will periodically send FCD messages to the `TseServerApp`.
+The server application is responsible for processing received FCD.
+
+The repository is structured into two main packages, `com.dcaiti.mosaic.app.fxd` and `com.dcaiti.mosaic.app.tse` trying to separate
+code that is concerned with the transmission and reception of messages (i.e., FCD) and code that is concerned with measuring the traffic
+state from the received FCD (i.e., TSE).
+
+### Usage
+>If this is your first time encountering Eclipse MOSAIC, please refer to the [Documentation](https://eclipse.dev/mosaic/docs/) and
+>[Tutorials](https://eclipse.dev/mosaic/tutorials/) to get a grasp of MOSAICs' core functionalities and working principles.
+
+**Mapping**
+
+To deploy the applications with your scenario, you need to adjust the *mapping_config.json* accordingly.
+Below is an exemplary configuration that you can adjust to your needs. Additionally, the bundled mapping file ([best-configuration/mapping/mapping_config.json](configs/best-scenario/mapping/mapping_config.json))
```

Comment:
the name of the link does not match the url

Reply:
fixed